src/Castle.Core/DynamicProxy/Contributors/InterfaceProxyWithoutTargetContributor.cs

```diff
@@ -93,5 +104,24 @@ private Type GetInvocationType(MetaMethod method, ClassEmitter emitter)
 				.Generate(emitter, namingScope)
 				.BuildType());
 		}
+
+		private MethodInfo CreateCallbackMethod(ClassEmitter emitter, MethodInfo methodInfo, MethodInfo methodOnTarget)
+		{
+			var targetMethod = methodOnTarget ?? methodInfo;
+			var callBackMethod = emitter.CreateMethod(namingScope.GetUniqueName(methodInfo.Name + "_callback"), targetMethod);
+
+			if (targetMethod.IsGenericMethod)
```

Comment:
There's currently no test for generic interface methods with default implementations. Might have to add at least one!